src/monitor/monitor.c

```diff
@@ -1342,18 +1347,26 @@ static void monitor_hup(struct tevent_context *ev,
     struct mt_ctx *ctx = talloc_get_type(private_data, struct mt_ctx);
     struct mt_svc *cur_svc;
 
-    DEBUG(SSSDBG_CRIT_FAILURE, "Received SIGHUP.\n");
+    DEBUG(SSSDBG_TRACE_FUNC, "Monitor received SIGHUP\n");
```

Comment:
I agree that shouldn't be `SSSDBG_CRIT_FAILURE`. But still this should be more prominent.
Let's use `SSSDBG_IMPORTANT_INFO` here